1914014.ts

```
@@ -0,0 +1,61 @@
+// A TypeScript file with intentional errors and bad practices
+
+// Function to add two numbers
+function addNumbers(a, b) {
+    return a + b; // Missing type annotations
+}
+
+console.log(addNumbers(5, "10")); // Mixing types, will concatenate instead of adding
+
+// Function to greet a user
+function greetUser(name: String): void { // Incorrect capitalization of 'string'
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)